src/Sidekick.Business/Apis/Poe/Trade/Data/Stats/Pseudo/PseudoStatDataService.cs

```diff
@@ -38,15 +34,13 @@ public async Task OnInit()
             {
                 logger.Information($"Pseudo stat service initialization started.");
 
-                var response = await client.GetAsync("https://www.pathofexile.com/api/trade/data/stats/");
-                var content = await response.Content.ReadAsStreamAsync();
-                var result = await JsonSerializer.DeserializeAsync<FetchResult<StatDataCategory>>(content, poeApiClient.Options);
+                var result = await poeApiClient.Fetch<StatDataCategory>();
```

Comment:
What was done here is that we always fetched the English stats and gather the Ids with regexes below in `InitGroups`. With this change here, we actually fetch in other languages so the regexes won't work anymore for any language other than English.
We wanted to avoid having to maintain other languages here (the regexes) since we only really need to do this once and compare ids like we do on `FillPseudo` in `StatDataService`.

Reply:
Ahh, yeah I missed that completely. I'll try to figure out a good solution for this

Reply:
I might be wrong, but you can just revert this part and it should work.

Reply:
The reason I changed it was to make use of the cached data from the PoE api and not make new requests each time the tests are run. Should be able to find a workaround though.

Reply:
I finally got around to fixing this. Ended up needing to make some changes to to the language provider to access the default language but I think it's fine for now. The language provider is a bit finicky work with at the moment and could probably do with a bit of cleaning up in the future.

Reply:
Cool, just message me when you feel it's ready.